packages/functions/test/integration/docs.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect, test } from 'vitest';
+import $ from 'tinyspawn';
+import path from 'path';
+
+(process.platform === 'win32' ? test.skip : test)(
+  'docs are up to date',
+  async () => {
+    const cwd = path.resolve(__dirname, '../');
+    await $('pnpm build:docs', { cwd });
+    const { stdout } = await $(`git status --short docs`);
```

Comment:
This only shows the current changes / working tree. Wouldn't this be always be empty when running in CI?

Reply:
I tested manually and if you forgot to commit docs it will throw an error!